db/errors/errors.go

```diff
@@ -0,0 +1,11 @@
+package errors
+
+import stderrors "errors"
+
+var (
+	// NoSchemasInManager is returned when there is no schemas in schema manager
+	NoSchemasInManager = stderrors.New("No schemas in Manager. Did you remember to load them?")
+
+	// ResourceNotFound is returned when requested resource was not found
+	ResourceNotFound = stderrors.New("resource not found")
```

Comment:
Obvious comments removed.
Golint shows it as an error. Unfortunately it doesn't have an option to remove that check.